# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esmart/hostfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/lib/esmart_container/Makefile.am.orig	2005-03-01 14:00:38.874801944 -0300
+++ ./src/lib/esmart_container/Makefile.am	2005-03-01 14:00:52.942663304 -0300
@@ -19,4 +19,4 @@
 				 esmart_container_util.c
 
 libesmart_container_la_LDFLAGS = -version-info 0:9:0
-libesmart_container_la_LIBADD = @imlib_libs@ @evas_libs@ -lm
+libesmart_container_la_LIBADD = @imlib_libs@ @evas_libs@ -lm -lltdl
